integration-tests/aqua/lib.aqua

```diff
@@ -163,3 +166,41 @@ func get_spell_errors(spell_id: string) -> AllErrorsResult:
 func noop():
 	on HOST_PEER_ID:
 		Op.noop()
+
+func set_spell_alias(alias: string, spell_id: string) -> Result:
+    result: ?Result
+    on HOST_PEER_ID:
+        peer_id <- get_peer_id()
+        on peer_id via HOST_PEER_ID:
+            try:
+                Srv.add_alias(alias, spell_id)
+                result <<- Result(success = true, error = "")
+            catch err:
+                result <<- Result(success = false, error = err.message)
+    <- result!
+
+func set_worker_spell_status(state: State, message: string) -> u64:
+    on HOST_PEER_ID:
+        peer_id <- get_peer_id()
+        on peer_id via HOST_PEER_ID:
+            set_status(state, message)
+            -- HACK: we need to return smth, because otherwise particle won't be processed at all
+            -- and we receive an error:
+            -- `particle_protocol::libp2p_protocol::upgrade: Error processing inbound ProtocolMessage: unexpected end of file
+            -- Location: /opt/actions-runner/_work/nox/nox/particle-protocol/src/libp2p_protocol/upgrade.rs:119:27`
+            ts <- Peer.timestamp_sec()
```

Comment:
Can you return `OK` instead of doing a syscall?

Reply:
sure

Reply:
But I think i should do smth like Op.identity("OK")

Reply:
No, `<- "OK"` should be enough

Reply:
I will check